project.json

```diff
@@ -1,6 +1,6 @@
 {
-  "serviceRole": "",
-  "serviceType": "",
+  "serviceRole": "exoservice-go",
+  "serviceType": "exoservice-go-type",
```

Comment:
Okay. Lets make it the same value as the serviceRole as the two are similar in most cases and only different when you are reusing services